spring-aot-test/src/test/java/org/springframework/aot/test/context/bootstrap/generator/TestContextAotProcessorTests.java

```diff
@@ -97,22 +98,41 @@ void processWritesContextLoaderMappingAtStandardLocation() {
 						"}");
 	}
 
+	@Test
+	void processWritesContextInitializersMapping() {
+		ContextBootstrapStructure structure = this.tester.process(
+				SampleApplicationTests.class, SampleApplicationAnotherTests.class, SimpleSpringTests.class);
+		assertThat(structure).contextBootstrapInitializer("TestContextBootstrapInitializer")
+				.removeIndent(1).lines().containsSubsequence(
+						"public static Map<String, Class<? extends ApplicationContextInitializer<?>>> getContextInitializers(",
+						"    ) {",
+						"  Map<String, Class<? extends ApplicationContextInitializer<?>>> map = new HashMap<>();",
+						"  map.put(\"org.springframework.aot.test.samples.app.SampleApplicationTests\", TestContextBootstrapInitializer0.class);",
+						"  map.put(\"org.springframework.aot.test.samples.app.SampleApplicationAnotherTests\", TestContextBootstrapInitializer0.class);",
+						"  map.put(\"org.springframework.aot.test.samples.simple.SimpleSpringTests\", SimpleSpringTestsContextInitializer.class);",
+						"  return map;",
+						"}");
+	}
+
 	@Test
 	void processInvokeTestNativeConfigurationRegistrar() {
 		ContextBootstrapStructure structure = this.tester.process(SampleApplicationTests.class);
 		assertThat(structure).hasResourcePattern("org/springframework/aot/test/samples/app/SampleApplication.class");
 	}
 
 	@Test
-	void processRegisterReflectionForContextLoadersMappingMethod() {
+	void processReflectionRegistrationForMappingMethods() {
```

Comment:
Please refrain from doing unrelated changes like this.

Reply:
Handled separately in 4787862c319fc609821922102f41e991abb5b1ef